views/partials/_whats-new.njk

```diff
@@ -0,0 +1,15 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<div class="app-whats-new" style="background-color:#f8f8f8; border-bottom:1px solid #b1b4b6">
+  <div class="app-width-container">
+    <div class="govuk-main-wrapper govuk-main-wrapper--l">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds-from-desktop">
+        <h2 id="whats-new" class="govuk-heading-l">What's new</h2>
+        <p class="govuk-body">20 May 2021: We’ve updated guidance in the cookie banner component. <a href="https://github.com/alphagov/govuk-design-system-backlog/issues/12#issuecomment-845097722" class="govuk-link">Read about this update</a>.</p>
+        <p class="govuk-body"><a href="/email-updates" class="govuk-link">Sign up to get updated by email</a>.</p>
```

Comment:
Trying to see if we can change the wording and line breaks to make the purpose behind the 'What's new' and email CTA a bit clearer.
```suggestion
        <h2 id="whats-new" class="govuk-heading-l">What's new</h2>
        <p class="govuk-body">20 May 2021: We’ve updated guidance in the cookie banner component.<br><a href="https://github.com/alphagov/govuk-design-system-backlog/issues/12#issuecomment-845097722" class="govuk-link">Read about this update</a>.</p>
        <p class="govuk-body"><a href="/email-updates" class="govuk-link">Sign up to get emails about the Design System</a>.</p>
```